src/docs/sphinx/advancedExamples/validationStudies/proppantSlotTest/Example.rst

```diff
@@ -0,0 +1,292 @@
+.. _ExampleProppantTest:
+
+
+####################################################
+Simulate Proppant Slot Test
+####################################################
+
+
+**Context**
+
+In this example, a simulation is built up to model proppant slot test. In this way, the implemented proppant model is validated by comparing numerical results with the corresponding experimental data. Furthermore, this calibrated proppant model can allow field engineers to customize stimulation design and optimize field operations in multiple engineering aspects `(Huang et al., 2021) <https://onepetro.org/ARMAUSRMS/proceedings/ARMA21/All-ARMA21/ARMA-2021-1248/467980>`__.
+
+
+**Input file**
+
+This example uses no external input files and everything required is
+contained within a single xml file that is located at:
+
+.. code-block:: console
+
+  src/coreComponents/physicsSolvers/multiphysics/benchmarks/Proppant-transport-validation-case.xml
+
+
+------------------------------------------------------------------
+Description of the case
+------------------------------------------------------------------
+
+`Chun et al. (2020) <https://www.sciencedirect.com/science/article/pii/S0920410519309441>`__ conducted slot tests on proppant transport with slickwater. As shown below, a 4 ft X 1 ft slot with 0.3 in (0.00762 m) gap width was constructed. Three fluid inlets with 0.5 in inner diameter were placed at the right side of the slot, which were three inches away from each other. One outlet was place on the top side to allow pressure relief. The other one was located on the left side acting as a fluid sink. In their test, to resemble a slickwater fracturing treatment, the proppant concentration was kept at 1.5 ppg and the viscosity of carrying fluid was approximately 1 cp. The slurry was mixed well and then injected into the flow channel at a constant injection rate (6 gpm). A simulation case with the same settings is built up to mimic this slot test. A vertical and impermeable fracture surface is assumed in this case, which eliminates the effect of fracture plane inclination and fluid leakoff. A static fracture with uniform aperture (0.3 in) is defined and fracture propagation is not involved. 30/50 mesh proppant is injected via the three inlets and is flowed through the slot for 30 seconds.
```

Comment:
Maybe in this description we could use the GEOSX unit system everywhere (meters, Pa.s, etc)?